### PR TITLE
Improve st2client homepage

### PR DESCRIPTION
**Pull request by blag**
This PR improves the PyPI description of st2client by doing the following:

* Removes the redundant `license` specifier argument to `setup()` (see the official documentation on it [here](https://packaging.python.org/specifications/core-metadata/#license), and why this is a problem [here](https://github.com/diyan/pywinrm/pull/284))
* Adds the content of `README.rst` as the long description for the st2client package on PyPI
* Adds additional URLs to the project, which I think will be rendered in the sidebar of the PyPI project page, giving the links more visibility for users looking at the st2client package on PyPI

### CODE REVIEW

**Review by nmaludy — COMMENTED** [review]
Should we set long_description_content_type to `text/x-rst`

https://packaging.python.org/guides/making-a-pypi-friendly-readme/

**Comment by blag** [comment]
According to [the docs](https://packaging.python.org/specifications/core-metadata/#description-content-type), that's not strictly necessary:

> If a Description-Content-Type is not specified, then applications should attempt to render it as text/x-rst; charset=UTF-8 and fall back to text/plain if it is not valid rst.

**Review by nmaludy — APPROVED** [review]
Great work, thank you!

**Comment by blag** [comment]
Thanks for the review!

File: st2client/setup.py [hunk 1]
```diff
@@ -63,5 +67,16 @@
         'console_scripts': [
             'st2 = st2client.shell:main'
         ]
+    },
+    project_urls={
```
Comment (armab) [inline st2client/setup.py:71]: That's a great list :+1:

Comment (blag) [reply st2client/setup.py:71]: Thanks!

**Pull request merged**